components/Storyblok/SbTypeform.tsx

```diff
@@ -32,7 +31,7 @@ export type SbTypeformProps = {
 };
 
 export const styles = {
-  container: 'cc flex flex-col place-items-center',
+  container: 'flex flex-col place-items-center',
```

Comment:
The default Container component `width="site"` setting adds the `cc` class, so I just remove both the `width="full"` below and the `cc` class here